.gitignore

```diff
@@ -9,6 +9,7 @@ bower_components/
 coverage/
 dist/
 kubernetes/
+tools/
```

Comment:
The number of auxiliary directories in our project grows every week. I'm wondering whether we should merge them into one and clean them on "gulp clean". What do you think? 

Not that current solution is bad. I'm just looking for improvements.